datasets/BBC_Hindi_NLI/BBC_Hindi_NLI.py

```diff
@@ -0,0 +1,151 @@
+# coding=utf-8
```

Comment:
could you move the files from the folder `BBC_Hindi_NLI` to `bbc_hindi_nli` ? We're using the snake case convention.
Could you also rename the dataset script accordingly ?